### ISSUE TITLE
Getting 403 error when login

### ISSUE THREAD

**Issue by aolmospersonal** [b31ngd3v/MyIGBot#21]
Hello, I have been working with ur API, and when I am trying to log in with correct user credentials I am getting 403 error, what does it mean?

**Comment by m2083902** [comment]
i think that does work any more or i dont know


În mie., 30 dec. 2020 la 00:18, k2sapex <notifications@github.com> a scris:

**Comment by b31ngd3v** [comment]
@m2083902 Hey why are you spreading rumours, this api is in working state. Works perfectly for me.

**Comment by aolmospersonal** [comment]
Yes it is working for other accounts, I don't know why in one account I am receiving this 403 error, I am putting correct credentials and haven't got 2fa so?

**Comment by Y7gn** [comment]
it's giving me 404
`from myigbot import MyIGBot

bot = MyIGBot('raedeneier', 'm123123123')

response = bot.story_view('b31ngdev')
print(response)  # if the response code is 200 that means ok`

**Comment by codecret** [comment]
from myigbot import MyIGBot

bot = MyIGBot('xx', 'xx')

response = bot.story_view('b31ngdev')
print(response)  # if the response code is 200 that means ok

**Comment by b31ngd3v** [comment]
this tool is dead.

**Comment by codecret** [comment]
Is there any tool else

Sent from my iPhone

On 15 Aug 2023, at 9:34 PM, Pramurta Sinha ***@***.***> wrote:

﻿

this tool is dead.
